### DISCUSSION TITLE
Systen hangs during loading using ventoy

### DISCUSSION THREAD

**Discussion by nslynx** [linuxdeepin/developer-center#9472] [Troubleshooting | 故障排查]
When trying to test a system, it hangs after getting this error in console:
failed to start deepin-login-sound.service
![20240628_015203](https://github.com/linuxdeepin/developer-center/assets/5699209/541b0668-38eb-458d-81b4-b34d5146e0c0)

**Comment by BLumia** [comment]
Are you trying to install deepin via a ventoy usb? If so, does this happen before starting the livecd session or after you install? Which deepin and ventoy version are you trying to use? Could you provide detailed info about how to reproduce this issue?

**Reply by nslynx** [reply]
Yes, I am trying to use ventoy usb. As it is a new version distro for me since 2017, I want to try it first. So, got this error with Deepin 23 RC2.
P.S. At first I have tried it using virtual box I had no such problems

As you understand, Linux Mint & Manjaro, for example, loaded without any problems

**Reply by BLumia** [reply]
Does this happen before/while starting the livecd session or after you install and while attempted to start the installed system?

**Reply by nslynx** [reply]
it is happened during starting livecd session, cause I wanted to try Deepin on my current PC, not Virtual Box

**Reply by BLumia** [reply]
cc @Zeno-sole

**Reply by BLumia** [reply]
https://github.com/linuxdeepin/developer-center/issues/9574 might related?

**Reply by nslynx** [reply]
Deepin 25 has just released. This bug is still here, May be it is specific for my Logi Z407 sound system?

**Reply by BLumia** [reply]
Not sure, sadly we don't have Z407 device to test :(

**Reply by nslynx** [reply]
I have disabled Z407 for test. As I have say, I can provide any additional info (I have created an issue)

**Reply by BLumia** [reply]
Ref: https://github.com/linuxdeepin/developer-center/issues/11806 so other people can find the issue.
